<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1181</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion Commande</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>671</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202, 202, 202);

border-color: rgb(15, 15, 15);</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter commande</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_commande</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NB_produit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix_total</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nb">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date_commande</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sup">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_mod">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modefier</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_rch">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rch_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_commande">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Historique">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>historique</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">*{


background-color: transparent;
border-image: url(C:/Users/khaled/Desktop/logo.png);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tri</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>TRIER PAR </string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion Biblioth√®ques </string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1181</width>
        <height>631</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../projet c++/Nuit-de-leau.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_bib">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>571</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rch_livre">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{


background-color: transparent;
border-image: url(C:/Users/khaled/Desktop/projet/projet/image/cherchbtn.png);

}

*:hover
{

background-color: transparent;


}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Livres </string>
      </property>
      <widget class="QLineEdit" name="id_Liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ref_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="desc_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>221</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateS_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateR_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="oui_liv">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>oui</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="non_liv_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Non</string>
       </property>
      </widget>
      <widget class="QLabel" name="l1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Livre</string>
       </property>
      </widget>
      <widget class="QLabel" name="l2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom livre</string>
       </property>
      </widget>
      <widget class="QLabel" name="l3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>R√©f Livre</string>
       </property>
      </widget>
      <widget class="QLabel" name="l4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description Livre</string>
       </property>
      </widget>
      <widget class="QLabel" name="l6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Recup√©ration</string>
       </property>
      </widget>
      <widget class="QLabel" name="l5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Sortie</string>
       </property>
      </widget>
      <widget class="QLabel" name="l7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Disponibilite</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Liv_ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Liv_modifier">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>270</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="liv_supprimer">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="rch_livre">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tri_livre">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{


background-color: transparent;
border-image: url(C:/Users/khaled/Desktop/projet/projet/image/triup.png);

}

*:hover
{

background-color: transparent;


}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{


background-color: transparent;
border-image: url(C:/Users/khaled/Desktop/projet/projet/image//print.png);

}

*:hover
{

background-color: transparent;


}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>FR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ANG</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tri</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>TRIER PAR </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>id_l</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM_L</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REF_LIVRE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>251</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Recherche </string>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom_livre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>R√©f_livre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cher1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="che2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="che3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="recherche">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
